ui-lib/src/main/java/co/electriccoin/zcash/ui/MainActivity.kt

```diff
@@ -133,6 +134,35 @@ class MainActivity : ComponentActivity() {
         }
     }
 
+    /**
+     * The body of this function is meat to be execute only while the app run on Firebase Test Lab or
+     * Google Play pre-launch report tests.
+     *
+     * It skips creating a new or restoring an existing wallet by persisting a wallet with a mock seed.
+     * As a result of this, it navigates the monkey test runner from onboarding screens right to the
+     * app home screen.
+     *
+     * @return true if the app is currently being messed with by a monkey test runner and the described
```

Comment:
`monkey test runner` I don't think this statement is entirely true as the monkey runner is separate from the test lab robo runner.